From 1477eb1fad92ac07cd057b3ecdb04edeeef9edba Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 4 Apr 2022 23:32:56 +0200
Subject: [PATCH] gst-libav: fix build on systems without C++ compiler

Fix the following build failure on systems without C++ compiler:

The following exception(s) were encountered:
Running "/nvmedata/autobuild/instance-9/output-1/host/bin/or1k-buildroot-linux-musl-g++ --version" gave "[Errno 2] No such file or directory: '/nvmedata/autobuild/instance-9/output-1/host/bin/or1k-buildroot-linux-musl-g++'"

Fixes:
 - http://autobuild.buildroot.org/results/8ac0ba5eaaf7571857b4d8cfabf1488d640dc59a

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/2112>

[Retrieved (and backported) from:
https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/1477eb1fad92ac07cd057b3ecdb04edeeef9edba]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 subprojects/gst-libav/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index fbfa3049afe..7da94b80d59 100644
--- a/meson.build
+++ b/meson.build
@@ -1,4 +1,4 @@
-project('gst-libav', 'c', 'cpp',
+project('gst-libav', 'c',
   version : '1.20.1',
   meson_version : '>= 0.60',
   default_options : [ 'warning_level=1',
-- 
GitLab

